src/pages/Product/ProductCard/ProductCard.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+import ProductIcon from './ProductIcon/ProductIcon';
+import ProductDescription from './ProductDescription/ProductDescription';
+
+import { AiOutlinePlus } from 'react-icons/ai';
+
+import './ProductCard.scss';
+
+export class ProductCard extends Component {
+  constructor() {
+    super();
+    this.state = {};
+  }
+
+  render() {
+    const { addCart, backgruonColor, handleCartButton } = this.props;
+
+    const { efficacy, name, icon, pillImage, description, quantity, price } =
+      this.props.productCard;
+
+    return (
+      <li className="ProductCard" style={{ backgroundColor: backgruonColor }}>
```

Comment:
인라인스타일 No No.... 이케이스는 그래도 약간 이해가 되니까 패스하도록 하겠습니다!
인라인 스타일 피하기 위해서 객체로 클래스네임 매핑할 수도 있습니다! 나중에 2차때 비슷한 상황 겪으시면 시도해보세요!

Reply:
객체 단축속성명이용해서 줄일 수 있습니다.
```suggestion
      <li className="ProductCard" style={{ backgroundColor}}>
```

Reply:
오와 !!!!! 여기까지 적용이 될거라고 생각하지 못했어요 👍🏻 주말에 좀더 연구해볼생각입니당!!!